src/System.Management.Automation/engine/remoting/fanin/OutOfProcTransportManager.cs

```diff
@@ -741,28 +741,11 @@ private void ProcessMessageProc(object state)
             }
         }
 
-        private const string GUIDTAG = "PSGuid='";
-        private const int GUID_STR_LEN = 36;        // GUID string: 32 digits plus 4 dashes
+        private const string SESSIONDMESSAGETAG = "PSGuid='00000000-0000-0000-0000-000000000000'";
 
-        private Guid GetMessageGuid(string data)
+        private bool IsSessionMessage(ReadOnlySpan<char> data)
```

Comment:
Just curious if we still need to pass as Span type.  We can just scan a normal string type, but I am not sure if it really makes a difference.

Reply:
:-) Yes, but we again should care about null to exclude a regression in future.

Reply:
I think if you worry that a null value may be passed to this method, the intent is more clear by having a null check here:
```
private bool IsSessionMessage(string data)
{
    if (data == null) { return false; }
    return data.IndexOf(SESSIONDMESSAGETAG, StringComparison.OrdinalIgnoreCase) > -1;
}
```
Or, maybe this method should just be directly inlined in `HandleOutputDataReceived` given it's just one line of code and not used anywhere else.

Reply:
I prefer having the private method since it makes clear what the purpose is.  Null check is not needed.

Reply:
Inlined the method.

Reply:
Oops, @PaulHigin sorry I did not see your comment.

Reply:
@PaulHigin Would it be OK to add a comment to the inlined code to make the purpose clear?
I don't mind having it a separate private method, just thought using `string data` should be sufficient.

Reply:
I assumed the compiler would inline but I am fine either way.  These are minor issues and overall I am happy with the changes.